document/core/exec/numerics.rst

```diff
@@ -1967,3 +1970,190 @@ Conversions
    \begin{array}{lll@{\qquad}l}
    \narrowu_{M,N}(i) &=& \satu_N(\signed_M(i))
    \end{array}
+
+.. _relaxed-ops:
+
+Relaxed operations
+~~~~~~~~~~~~~~~~~~
+
+The result of *relaxed* operators are *host-dependent*, because the set of
+possible results may depend on properties of the host environment (such as
+hardware). Technically, each such operator produces a fixed-size *list of sets*
+of allowed values.  For each execution of the operator in the same environment,
+only values from the set at the same position in the list are returned, i.e.,
+each environment globally chooses a fixed projection for each operator.
+
+.. note::
+   Each operator can be thought of as a family of operations that is fixed to
+   one particular choice by the execution environment. The fixed operation
+   itself can still be non-deterministic or partial.
+
+
+.. _op-relaxed_madd:
+
+:math:`\relaxedmadd_N(z_1, z_2, z_3)`
+.....................................
+
+Relaxed multiply-add (madd) allows for fused or unfused results. The function :math:`fma` is
+defined by |IEEE754|_ (Section 5.4.1) as *fusedMultiplyAdd*.
```

Comment:
Aren't we still missing a definition for this function?

Reply:
Done, ptal.